msal/application.py

```diff
@@ -1552,6 +1655,19 @@ def acquire_token_interactive(
 
             New in version 1.15.
 
+        :param int window:
+            OPTIONAL. If your app is a GUI app running on modern Windows system,
+            and your app opts in to use broker,
+            you are recommended to also provide its window handle,
+            so that the sign in UI window will properly pop up on top of your window.
+
+        :param boolean enable_msa_passthrough:
```

Comment:
Maybe we want to add its default value which is False I gues?

Reply:
Thanks. Added one more line in the latest rebase.